docs/pages/guides/authentication.mdx

```diff
@@ -1940,6 +1942,115 @@ export default function App() {
 </Box>
 {/* End Uber */}
 
+{/* Start Calendly*/}
```

Comment:
Same as above. On this page, we follow alphabetical order so let' stick to that. Please remove this line from here and add it before "Cognito".